.github/workflows/testing.yml

```diff
@@ -395,7 +306,7 @@ jobs:
              /home/dev/taichi/.github/workflows/scripts/unix_test.sh
         env:
           PY: py38
-          TI_WANTED_ARCHS: 'cuda,vulkan,opengl'
+          TI_WANTED_ARCHS: 'cpu,cuda,vulkan,opengl'
```

Comment:
just to make sure we don't accidentally reduce coverage, shall we keep `cc` in this list as well? 
(which should be removed in a separate PR along with cc backend if we want. )

Reply:
Added `cc` to the list